server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CICSParser.g4

```diff
@@ -1046,7 +1046,7 @@ idms_only_words
     | WCC | WHITE | WITHIN | YELLOW
     ;
 
-cicsWord: NONNUMERICLITERAL | NUMERICLITERAL | integerLiteral | generalIdentifier | cobolWord | cics_cobol_intersected_words;
+cicsWord: NONNUMERICLITERAL | NUMERICLITERAL | integerLiteral | generalIdentifier | cics_cobol_intersected_words;
```

Comment:
Not sure if removal of cobolWord will cause a fail, at least on some tests?

Reply:
`cobolWord` is indirectly covered by the `generalIdentifier`